src/module.ts

```diff
@@ -179,18 +179,20 @@ export default defineNuxtModule<ModuleOptions>({
 
     // Include CSS file in project css
     let resolvedCss: string
+    const { resolve: relativeResolve } = createResolver(import.meta.url)
+
     if (typeof cssPath === 'string') {
       if (existsSync(cssPath)) {
         logger.info(`Using Tailwind CSS from ~/${relative(nuxt.options.srcDir, cssPath)}`)
         resolvedCss = cssPath
       } else {
         logger.info('Using default Tailwind CSS file from runtime/tailwind.css')
         // @ts-ignore
-        resolvedCss = createResolver(import.meta.url).resolve('runtime/tailwind.css')
+        resolvedCss = relativeResolve('runtime/tailwind.css')
       }
     } else {
       logger.info('No Tailwind CSS file found. Skipping...')
-      resolvedCss = createResolver(import.meta.url).resolve('runtime/empty.css')
+      resolvedCss = relativeResolve('runtime/empty.css')
     }
     nuxt.options.css = nuxt.options.css ?? []
 
```

Comment:
(Next line):

```ts
const resolvedNuxtCss = await Promise.all(nuxt.options.css.map((p: any) => resolvePath(p.src ?? p)))
```

While I've been planning for some type-improvements, I also want to ask about this part -
`nuxt.options.css` is of type `string[]` for both [Nuxt 3](https://nuxt.com/docs/api/configuration/nuxt-config#css) and [Nuxt 2](https://nuxtjs.org/docs/configuration-glossary/configuration-css), so lets not consider objects?

Reply:
Actually in Nuxt 2 we could have `src` and `type` before we smartly guess the type from the file extension